tanstack-table/src/main/scala/lucuma/react/table/HTMLTableRenderer.scala

```diff
@@ -228,7 +228,7 @@ trait HTMLTableRenderer[T]:
                 <.tr(
                   TbodyTrSubcomponentClass,
                   TbodyTrEvenClass.when((index + indexOffset) % 2 =!= 0),
-                  ^.key := s"${row.id}-subcomponent"
+                  ^.key := s"${row.id}-2"
```

Comment:
why `2`?

Reply:
It's just shorter than `subcomponent`.

The main row now has `-1` suffix and the subcomponent row `-2` (and the enclosing `Fragment` no suffix).